03-Raster-Calculations-In-R.Rmd

```diff
@@ -133,6 +149,25 @@ plot(DSM_HARV,
      main="Digital Surface Model (Elevation)\n Harvard Forest")
 ```
 
+Notice that the elevation range for these two maps (DSM and DTM) is different. 
+Why?  Is there any reason to question our data because of this elevational
```

Comment:
Exactly!!!  I want to make sure people realized how exactly the DTM and DSM are different even though they are of the same plot of land.  Just making sure people understand what the layers are.  I fear that too often newbie's think "elevation" map not ground/terrain vs surface maps.